app/main.py

```diff
@@ -1 +1,55 @@
-# write your code here
+from time import sleep 
+
+
+class SoftwareEngineer:
+    def __init__(self, name: str, skills: list) -> None:
+        self.name = name
+        self.skills = skills
+    
+    def learn_skill(self, skill: str) -> None:
+        self.__class__.skills.append(skill)
```

Comment:
The use of `self.__class__.skills.append(skill)` is incorrect. `skills` should be an instance attribute, so you should use `self.skills.append(skill)` instead.